module/utils.mjs

```diff
@@ -49,7 +49,8 @@ export function isValidDieModifier(mod) {
     reroll: /rr?([0-9]+)?([<>=]+)?([0-9]+)?/i,
     explode: /xo?([0-9]+)?([<>=]+)?([0-9]+)?/i,
     minimum: /(?:min)([0-9]+)/i,
-    maximum: /(?:max)([0-9]+)/i
+    maximum: /(?:max)([0-9]+)/i,
+    keep: /k([hl])?([0-9]+)?/i
```

Comment:
Are we missing 'drop lowest' modifiers as well in that case?

Reply:
`unsoluble-discord-avatar.jpg` 😩

Reply:
I added drop as well as count (for "cs" and "cf"). That should be more than what is needed for most standard games of dnd. It would be handy if these regular expressions were exposed directly somewhere in the Roll or Term classes.

Reply:
Well, core doesn't really match individual modifiers with regular expressions like this.